php/db.php

```diff
@@ -73,6 +76,103 @@ function GetDataForTable($tabName){
 	return $tabData;
 }
 
+function MigrateDatabase() {
+	global $dbConn, $ip, $userAgent, $dbVersion;
+	$migrationsDir = "SQL/versions/";
+
+
+	StartTimer("MigrateDatabase");
+
+	// Check our current database version
+	$sql = "SELECT config_value FROM config WHERE config_key = 'DATABASE_VERSION';";
+	$data = mysqli_query($dbConn, $sql);
+	
+	$config_result = mysqli_fetch_array($data);
+
+	if ($config_result == NULL) {
+		die("Unable to determine the database version. Please add the config value DATABASE_VERSION before running");
+	}
+
+	$currentDatabaseVersion = intval($config_result[0], 10);
+
+	// Check and see if we need to migrate.
+	if ($dbVersion < $currentDatabaseVersion) {
+		die('Database version is out of date with code version');
```

Comment:
I read the text as "You need to update the database to bring it up to the code version", but the reverse is true. Can this be changed to something like "Database version is newer than the site code, please update the site code", or something similar?

Reply:
Good catch, 100% agree